site/topics/layout/bus.rst

```diff
@@ -73,6 +73,62 @@ Tri-State Logic
 Connecting Components
 =====================
 
+.. figure:: ram_register_schematic.png
+    :width: 500 px
+    :align: center
+
+    RAM and a register connected to various busses. This configuration allows for inputting data into the bus from an
+    external source (the input buttons), controlling the memory address used in RAM, and controlling when the RAM and
+    register components are inputting/outputting data.
+
+
+* Above is a RAM component connected to a register component via a data bus
+* With this configuration, data can be inputted into the system via the input buttons
```

Comment:
..data can be _input_ into... -- sounds more correct to me; or maybe ..data can be fed into the system via the input.. -- since _input_ is already in the sentence